src/BattleServer/moves.cpp

```diff
@@ -8163,14 +8176,15 @@ struct MMZSupernova : public MM
 
     static void zm(int, int, BS &b) {
         if(b.terrain != BS::PsychicTerrain) {
-            //No terrain extender cause you can only hold 1 item!
```

Comment:
Also Hackmons still only allows 1 item to be held... You can't remove a Z Crystal either